src/components/product-grid/product-grid.module.scss

```diff
@@ -13,10 +13,10 @@
 
 .clearFiltersButton {
     margin-top: 32px;
-    cursor: pointer;
+    padding: 0;
 }
 
-.clearFiltersButton:hover {
+.clearFiltersButton:not(:disabled):hover {
```

Comment:
This is because of specificity that is coming from the common variants

Reply:
I think it's wrong to add the class `.button` to things like icon buttons that don't have the appearance of a regular button. Because then any changes to the styling of the regular button could easily leak into those  buttons.

Also overrides like `.clearFiltersButton:not(:disabled):hover` would not be necessary if this button simply didn't include the class `.button`

Reply:
I can add a specific class with a something like `iconButton`?

Reply:
I think `.iconButton` and `.linkButton` would be nice, we could provide hover and disabled styles for those.

Then for example this `clearFilters` button could be styled like:

```jsx
<button className={classNames(styles.clearFiltersButton, 'linkButton')}>
```

```diff
.clearFiltersButton {
    margin-top: 32px;
-   padding: 0;
-   cursor: pointer;
}

-.clearFiltersButton:not(:disabled):hover {
-    color: rgba(23, 17, 13, 0.7);
-}
```

Reply:
replaced